src/app/features/jobs/pages/job-details/job-details.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { JobService } from '../../service/job.service';
+import { Observable } from 'rxjs';
+import { Job } from '../../models/job';
+
+@Component({
+  selector: 'app-job-details',
+  templateUrl: './job-details.component.html',
+  styleUrl: './job-details.component.scss'
+})
+export class JobDetailsComponent implements OnInit {
+
```

Comment:
Remove constructor and add them here with inject methods.

Example: private readonly route = inject(ActivatedRoute)....